app/src/renderer/components/common/TextBlock.vue

```diff
@@ -1,4 +1,6 @@
 <template>
+  <!-- TODO:SECURITY try to add a malicious block and see how is rendered, like <script>alert('danger')</script> -->
```

Comment:
is there an issue for this ?

Reply:
Not yet but I was thinking of tagging all the possible security holes as TODO:SECURITY, so when we will have an internal security audit some possible weak points are highlighted

Reply:
opened 👍